README.md

```
@@ -131,9 +131,9 @@ The emphasis on privacy, security, and community engagement makes CinemataCMS pa
 
 ## Installation
 The instructions below have been tested on Ubuntu 22.04. Make sure no other services are running on the system (specifically no Nginx or PostgreSQL), as the installation script will install and configure them.
- 
+
 **Production install (as root):**
- 
+
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)